core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -70,14 +70,24 @@
     </li>
 
     {% if username %}
-      <li ng-if="isLargeScreen" class="dropdown" popover-is-open="postTutorialHelpPopoverIsShown"
+    <div class="oppia-help-dropdown nav navbar-nav oppia-navbar-nav pull-right ng-cloak">
```

Comment:
The additional class should be in the line below, not this one. (We're manipulating the popover directly, not this class)